app/src/main/java/edu/uc/groupproject/covid19tracker/ui/main/MainViewModel.kt

```diff
@@ -32,54 +32,58 @@ class MainViewModel : ViewModel() {
 
```

Comment:
Changed many of the var's to val's because they are never changed after they are first initialized.

Reply:
Good suggestion, though I believe this was covered in a previous code review.